pyronear/__init__.py

```diff
@@ -1,3 +1,4 @@
 from pyronear import datasets
+from pyronear import models
 from pyronear import utils
 from .version import __version__
```

Comment:
This should not fail your pep8 checks ?

Reply:
How so?

Reply:
No newline at the end of file

Reply:
Oh, the branch was created before pep8 was introduced. So when merging into master (where that end of line has been added), it will be fine.